### PR TITLE
sysconfig/motr: improve perf params documentation

### PR DESCRIPTION
**Pull request by andriytk**
# Problem Statement
Perf-related params are poorly documented.

# Design
Add some more documentation to such performance-related
parameters as MOTR_M0D_MIN_RPC_RECVQ_LEN and
MOTR_M0D_IOS_BUFFER_POOL_SIZE.

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by huanghua78 — APPROVED** [review]
very good explanation, and it will definitely help.

File: scripts/install/etc/sysconfig/motr [hunk 1]
```diff
@@ -63,9 +63,17 @@ MOTR_M0T1FS_MOUNTPOINT='/mnt/m0t1fs'
...
 
 # Minimum rpc receive queue length.
+# Packets from clients will be dropped if the queue is not big enough,
+# and this might affect the performance dramatically. Increase this
+# (for example, to 512) to avoid packets drops on receive.
 #MOTR_M0D_MIN_RPC_RECVQ_LEN=16
 
-# ioservice buffer pool configuration
+# ioservice buffer pool configuration.
+# This parameter relates to the above MOTR_M0D_MIN_RPC_RECVQ_LEN, it
```
Comment (madhavemuri) [inline scripts/install/etc/sysconfig/motr:72]: This is not right, one RPC message may require many zero copy buffers.
Both are independent.

Comment (andriytk) [reply scripts/install/etc/sysconfig/motr:72]: They are independent, but they are related also, aren't they? And I tried to explain, how they are related. Please, suggest a better text which will be more clear.

Comment (madhavemuri) [reply scripts/install/etc/sysconfig/motr:72]: They are not related.
IOS and SNS buffers are independent, impact IO Writes/Reads and SNS IO only.

File: scripts/install/etc/sysconfig/motr [hunk 2]
```diff
@@ -63,9 +63,17 @@ MOTR_M0T1FS_MOUNTPOINT='/mnt/m0t1fs'
...
+# controls the number of spare buffers on receive for 0-copy. Set it
+# to the value which is about twice more than MOTR_M0D_MIN_RPC_RECVQ_LEN.
+# (One RPC msg may have several 0-copy transfers. For example, 2MB
+# unit write request will have two 0-copy , as LNet max frame size is 1MB.)
```
Comment (madhavemuri) [inline scripts/install/etc/sysconfig/motr:76]: Let's not add Lnet, with Libfab there is no restriction of 1MB.

File: scripts/install/etc/sysconfig/motr (hunk 1 above)
Comment (andriytk) [reply scripts/install/etc/sysconfig/motr:72]: Ok, I see it is confusing, so let me drop this "related" note.

File: scripts/install/etc/sysconfig/motr (hunk 2 above)
Comment (andriytk) [reply scripts/install/etc/sysconfig/motr]: Ok, dropped.

Comment (andriytk) [reply scripts/install/etc/sysconfig/motr]: Ok, dropped.

Comment (andriytk) [reply scripts/install/etc/sysconfig/motr]: Ok, dropped.

**Review by madhavemuri — APPROVED** [review]

**Comment by madhavemuri** [comment]
@gauravchaudhari02 : Can you check why k8s deployment job is not triggered for it ?

**Pull request merged**